{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--b672efb9-372f-4649-a7d2-ec0704cf11b5",
    "created": "2020-05-26T20:30:02.295135Z",
    "modified": "2020-05-26T20:30:02.295135Z",
    "name": "CVE-2019-16227",
    "description": "An issue was discovered in py-lmdb 0.97. For certain values of mn_flags, mdb_cursor_set triggers a memcpy with an invalid write operation within mdb_xcursor_init1. NOTE: this outcome occurs when accessing a data.mdb file supplied by an attacker."
}